flyteidl2/workflow/run_definition.proto

```diff
@@ -410,3 +410,9 @@ message ActionSpec {
   // group this action belongs to, if applicable.
   string group = 8;
 }
+
+enum RunSource {
+  RUN_SOURCE_UNSPECIFIED = 0;
+  RUN_SOURCE_USER = 1;
```

Comment:
Can we break this apart into CLI, CODE, UI ? I think this is going to be a useful metric to show...